packages/itwinui-css/src/slider/classes.scss

```diff
@@ -6,6 +6,13 @@
   @include iui-slider-component-container;
 }
 
+// TODO: See if this is the correct place for non-iui related selectors
+#demo-default-vertical,
+#demo-range-vertical {
+  display: flex;
+  gap: 2rem;
+}
+
```

Comment:
After running `yarn test`, I realized that there is a `<section>` in tabs.html with the same id as the sliders. `#demo-default-vertical`. Hence, `tabs.html` is also affected by the selected lines above and so the backstop test for `tabs.html` also fails.

What do you recommend? Here are some solutions I can think of:
* Change the id of the `sliders.html`'s `<section>` to something else, like `#demo-default-vertical-slider`?
* Remove the selected lines of code above. Allow the vertical slides to be arranged vertically instead of horizontally.

Reply:
Thanks, moved that code from `classes.scss` to `slider.html`. The failing backstop test now passes